docs/Actions/apply-tokens.md

```diff
@@ -0,0 +1,165 @@
+---
+id: apply-tokens
+title: Apply Tokens
+sidebar_label: Apply Tokens
+---
+
+> Audience: [`Citizen Developer`](/docs/audience#citizen-developers)<br/>
+> Skill Prerequisites: `Variables`
+
+This actions executes multiple expressions and applies tokens on those expressions and stores it in another (or the same) token(s). Note that this presents a security risk when the expression has content from untrusted users. 
+
+## `Typical Use Cases`
+
+- Execute a Variable expression that returns an token that needs to be reevaluated
+- Apply tokens inside a text from various sources (eg: other actions, form fields, API input parameters)
+
+## `Don't use it to`
+
+- Concatenate 2 tokens - Use [Inject Data](/docs/Actions/inject-data.md)
+- Overwrite a token - Use [Inject Data](/docs/Actions/inject-data.md)

```

Comment:
Got it. But why shouldn't I use this action instead?

Reply:
There is no reason. I was trying to encourage people to try more actions. Actually Inject Data, Execute Token and Apply Tokens can be one action.